packages/components/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "es2021",
```

Comment:
The table is kinda hard to read, but it seems to me that all browsers and recent versions of Node support ES2022 already.

https://compat-table.github.io/compat-table/es2016plus/